examples/speaker_recognition/voxceleb_eval.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import sys
+
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import brentq
+from sklearn.metrics import roc_curve
+from tqdm import tqdm
+
+
+"""
+This script faciliates to get EER % based on cosine-smilarity 
+for Voxceleb dataset.
+
+Args:
+    trial_file str: path to voxceleb trial file
+    emb : path to numpy file of embeddings (generated from spkr_get_emb.py)
+    manifest: path to test manifest file that contains path of audio files which is used in spkr_get_emb.py
+    Note: order of audio files in manifest file should match the embeddings
+"""
+
+
+def get_labels(manifest):
+    lines = open(manifest, 'r').readlines()
```

Comment:
minor nit: you could use `with open(...)` to open files or make sure you close them. Should not really matter much in python, but still :)

Reply:
Done